Tools/Create-Azure-Sentinel-Solution/input/Solution_CohesitySecurity.json

```diff
@@ -0,0 +1,21 @@
+{
+    "Name": "CohesitySecurity",
+    "Author": "Microsoft - support@microsoft.com",
```

Comment:
@yinghuang123, please change the author details, thanks.

Reply:
The file was removed